packages/curve-ui-kit/src/features/layout/store.ts

```diff
@@ -27,22 +25,14 @@ interface LayoutState {
 
 interface LayoutActions {
   setLayoutWidth: (pageWidthClassName: PageWidthClassName) => void
-  setLayoutHeight: (key: keyof LayoutHeight, value: number) => void
+  setNavHeight: (value: number) => void
   updateShowScrollButton: (scrollY: number) => void
   setScrollY: (scrollY: number) => void
   setPageVisible: (visible: boolean) => void
 }
 
-const DEFAULT_LAYOUT_HEIGHT: LayoutHeight = {
-  globalAlert: 0,
-  mainNav: 0,
-  secondaryNav: 0,
-  footer: 0,
-}
-
 const DEFAULT_STATE: LayoutState = {
-  height: DEFAULT_LAYOUT_HEIGHT,
-  navHeight: 0,
+  navHeight: 96, // Default height for desktop
```

Comment:
I guess we're sacrifing some clarity on how the building blocks of the navHeight with a simple 'magic' number. I don't mind either way tbh, simplifies everything by a lot

Reply:
This is just the initial value, will add a comment